src/app.js

```diff
@@ -2,20 +2,8 @@
 
 const _ = require('lodash')
 const co = require('co')
-const metadata = require('./controllers/metadata')
```

Comment:
Not at the moment. Pairs is only used by the visualization, and I removed it in https://github.com/interledgerjs/five-bells-visualization/pull/35. Metadata isn't needed anymore since there are no URLs.

@justmoon do you think any of these features should be replicated with `sendMessage`/`incoming_message`, or are they no longer needed?